packages/inventory-insights/src/Insights.js

```diff
@@ -163,7 +164,11 @@ class InventoryRuleList extends Component {
                         { title: <div> {rule.description}</div> },
                         {
                             title: <div key={key}>
-                                {moment(rule.publish_date).fromNow()}
+                                <Tooltip key={key} position={TooltipPosition.bottom} content={
```

Comment:
Please do not use moment.js as it bring a ton of unnecessary code. You don't really need it since `DateFormat` does everything for you
```JSX
title: <div key={key}>
  <DateFormat date={rule.publish_date} />
</div>
```